src/txn/txn_ckpt.c

```diff
@@ -287,57 +281,24 @@ static int
 __checkpoint_apply_operation(
   WT_SESSION_IMPL *session, const char *cfg[], int (*op)(WT_SESSION_IMPL *, const char *[]))
 {
-    WT_CONFIG targetconf;
-    WT_CONFIG_ITEM cval, k, v;
+    WT_CONFIG_ITEM cval;
     WT_DECL_ITEM(tmp);
     WT_DECL_RET;
-    bool ckpt_closed, named, target_list;
-
-    target_list = false;
+    bool ckpt_closed, named;
 
     /* Flag if this is a named checkpoint, and check if the name is OK. */
     WT_RET(__wt_config_gets(session, cfg, "name", &cval));
     named = cval.len != 0;
-    if (named)
+    if (named) {
         WT_RET(__checkpoint_name_ok(session, cval.str, cval.len, false));
-
-    /* Step through the targets and optionally operate on each one. */
-    WT_ERR(__wt_config_gets(session, cfg, "target", &cval));
-    __wt_config_subinit(session, &targetconf, &cval);
-    while ((ret = __wt_config_next(&targetconf, &k, &v)) == 0) {
-        if (!target_list) {
-            WT_ERR(__wt_scr_alloc(session, 512, &tmp));
-            target_list = true;
-        }
-
-        if (v.len != 0)
-            WT_ERR_MSG(session, EINVAL, "invalid checkpoint target %.*s: URIs may require quoting",
-              (int)cval.len, (char *)cval.str);
-
         /* Some objects don't support named checkpoints. */
-        if (named)
-            WT_ERR(__checkpoint_name_check(session, k.str));
-
-        if (op == NULL)
-            continue;
-        WT_ERR(__wt_buf_fmt(session, tmp, "%.*s", (int)k.len, k.str));
-        if ((ret = __wt_schema_worker(session, tmp->data, op, NULL, cfg, 0)) != 0)
-            WT_ERR_MSG(session, ret, "%s", (const char *)tmp->data);
+        WT_ERR(__checkpoint_name_check(session));
     }
-    WT_ERR_NOTFOUND_OK(ret, false);
-
-    if (!target_list && named)
```

Comment:
It is still there. There were two ifs before with different if conditions: old lines 301-302 and 329-331. Now there is only one if since the if conditions are the same: new lines 292-296.

Reply:
Got it - thank you. The structure threw me.